omod/src/main/webapp/addLabTestAttributeType.jsp

```diff
@@ -798,11 +798,11 @@ legend.scheduler-border {
 				 var dataTypeConfig = document.getElementById('datatypeConfig').value;
 				     if(dataTypeOp == "org.openmrs.customdatatype.datatype.ConceptDatatype"){
 				    	 if(dataTypeConfig != "" && dataTypeConfig != null ){
-				        		if(checkConceptExistent(dataTypeConfig)){
+				        		if(checkConceptExists(dataTypeConfig)){
 				        			 document.getElementById("datatypeconfig").style.display= 'none';
 				        		 }else{	        			 
 				        			  document.getElementById("datatypeconfig").style.display= 'block';	
-				      				  document.getElementById('datatypeconfig').innerHTML = "Concept Id  does not exist in concept dictionary";
+				      				  document.getElementById('datatypeconfig').innerHTML = "Either concept does not exist in concept distionary or please enter valid concept UUID";
```

Comment:
Rephrase to "Unable to get reference object". There are multiple possibilities of error.

Reply:
There are only two possibilities of this error. And the suggested error message is not user friendly.